src/TLuaInterpreter.cpp

```diff
@@ -15741,14 +15657,81 @@ bool TLuaInterpreter::callEventHandler(const QString& function, const TEvent& pE
             lua_pushnumber(L, qME->angleDelta().y());
             lua_setfield(L, -2, QStringLiteral("angleDeltaY").toUtf8().constData());
 
-            error = lua_pcall(L, pE.mArgumentList.size() + 1, LUA_MULTRET, 0);
+            error = lua_pcall(L, 1, LUA_MULTRET, 0);
             break;
         }
         }
     } else {
-        error = lua_pcall(L, pE.mArgumentList.size(), LUA_MULTRET, 0);
+        error = lua_pcall(L, 0, LUA_MULTRET, 0);
+    }
+
+    if (error) {
+        std::string err = "";
+        if (lua_isstring(L, -1)) {
+            err += lua_tostring(L, -1);
+        }
+
+        QString function = "setLabelCallback";
+        QString name = "label callback event";
+        logError(err, name, function);
+        if (mudlet::debugMode) {
+            TDebug(QColor(Qt::white), QColor(Qt::red)) << "LUA: ERROR running script " << function << " (" << function << ")\nError: " << QString::fromUtf8(err.c_str()) << "\n" >> 0;
+        }
+    }
+
+    lua_pop(L, lua_gettop(L));
+    return !error;
+}
+
+// No documentation available in wiki - internal function
+bool TLuaInterpreter::callEventHandler(const QString& function, const TEvent& pE)
+{
+    if (function.isEmpty()) {
+        return false;
+    }
+
+    lua_State* L = pGlobalLua;
+
+    int error = luaL_dostring(L, QStringLiteral("return %1").arg(function).toUtf8().constData());
+    if (error) {
+        std::string err;
+        if (lua_isstring(L, 1)) {
+            err = "Lua error:";
```

Comment:
Though this is a copy of the previous code there is a needed space missing here after the `:` in the string.